spk/calibrephp/src/installer.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+# Package
+PACKAGE="calibrephp"
+DNAME="CalibrePHP"
+PACKAGE_NAME="com.synocommunity.packages.${PACKAGE}"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+WEB_DIR="/var/services/web"
+DB_FILE="${WEB_DIR}/${PACKAGE}/app/Config/database.php"
+USER="$([ $(grep buildnumber /etc.defaults/VERSION | cut -d"\"" -f2) -ge 4418 ] && echo -n http || echo -n nobody)"
+
+preinst ()
+{
+	exit 0
+}
+
+postinst ()
+{
+	# Link
+	ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+	# Install the web interface
+	cp -pR ${INSTALL_DIR}/share/${PACKAGE} ${WEB_DIR}
+
+	# Set permissions
+	chmod -R ga+w ${WEB_DIR}/${PACKAGE}/app
+
+	if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+		# Configure open_basedir
+		if [ "${USER}" == "nobody" ]; then
+			echo -e "<Directory \"${WEB_DIR}/${PACKAGE}\">\nphp_admin_value open_basedir ${WEB_DIR}/${PACKAGE}:${wizard_calibre_dir:=/volume1/calibre/} \n</Directory>" > /usr/syno/etc/sites-enabled-user/${PACKAGE}.conf
+		else
+			echo -e "[PATH=${WEB_DIR}/${PACKAGE}]\nopen_basedir = ${WEB_DIR}/${PACKAGE}:${wizard_calibre_dir:=/volume1/calibre/}" > /etc/php/conf.d/${PACKAGE_NAME}.ini
+		fi
+
+		# Set metadata path
+		if [ -f "${wizard_calibre_dir:=../}metadata.db" ]; then
```

Comment:
This should probably be the same defaultValue as you specify in the wizard? Same on L40.